frontend/src/main/scala/bloop/bsp/BloopBspServices.scala

```diff
@@ -662,6 +668,56 @@ final class BloopBspServices(
     }
   }
 
+  def jvmTestEnvironment(
+      params: bsp.JvmEnvironmentParams
+  ): BspEndpointResponse[bsp.JvmEnvironmentResult] = {
+    ifInitialized(None) { (state: State, logger: BspServerLogger) =>
+      mapToProjects(params.targets, state) match {
+        case Left(error) =>
+          logger.error(error)
+          Task.now((state, Left(JsonRpcResponse.invalidRequest(error))))
+
+        case Right(projects) =>
+          val environmentEntries: List[Either[String, JvmEnvironmentEntry]] = for {
+            (id, project) <- projects.toList
+            dag = state.build.getDagFor(project)
+            fullClasspath = project.fullClasspath(dag, state.client).map(_.toString)
+            environmentVariables = state.commonOptions.env.toMap
+            workingDirectory = state.commonOptions.workingDirectory
```

Comment:
I think it would be fine to silently ignore non-JVM targets here. It's expected that non-JVM targets have no JVM execution environment.
```suggestion
            workingDirectory = state.commonOptions.workingDirectory
            config <- project.platform match {
              case Platform.Jvm(config, _, _) => Some(config)
              case _ => None
            }         
```

Reply:
Done